### ISSUE TITLE
Render only after update/create?

### ISSUE THREAD

**Issue by labcenter2020** [pdmfc/nova-info-card#12]
Hi there,

I've just come across your package and it looks great and very similar to what I was trying to create. 

Is it possible to register the cards so its only shown after creating or updating a resource?

I was hoping to show some form of "Please wait 5 minutes for this to take effect" message.

I started going down the resource tool route but it seemed a bit convoluted to show a simple message. I don't suppose anyone has done this before or the "Request $request" variable can detect if the resource has been created or updated?

**Comment by skoyah** [comment]
Hi there @labcenter2020 

Thanks for your feedback. I would like to know a bit more about your use-case. 

So if I understood correctly, you want to show the card right after creating or updating the resource. Is that right?

In that case, if the user leaves the detail view, will you still want to be able to show the card for a certain period of time?

What you can do, is to use the `created_at` and `updated_at` timestamps to conditionally render the card using the `canSee()` method.

Would that be a viable solution?

**Comment by labcenter2020** [comment]
Hi @skoyah,

Thanks for the prompt reply. Sorry I didn't seem to get a notification. 

Yep, that's right! I could see it working either way really. Either a flash or something which persists for a certain time would work nicely.  

We currently have an observer class on a few resources which sends some data to an API so we could ideally do with a popup/message which says something about it taking a few minutes to take effect. 

Thanks for the suggestion that sounds like a very viable option and is likely the one I will go with for the time being.

I actually put together a few bits I tested and posted on https://github.com/laravel/nova-issues/issues/2960

The only downside of using the canSee/card solution I found was if the user was using the "Update & Continue Editing" or "Create and Add Another" buttons. 

I think the only way to get around those would be by having some emit / event bus trigger and a global modal/popup type component. I only have a limited knowledge of Vue JS though so there might be a more efficient way. 

Thanks for your help!
